target_goal:
  standby: {pose: [-2.113, -0.348, 0.0], orientation: [0.0, 0.0, 0.710, 0.704]}
  pick: {pose: [-2.478, 9.483, 0.0], orientation: [0.0, 0.0, 0.018, 1.0]}
  place: {pose: [-0.542, -0.269, 0.0], orientation: [0.0, 0.0, 1.0, 0.018]} 
  pose1: {pose: [-2.478, 9.483, 0.0], orientation: [0.0, 0.0, 0.018, 1.0]}
  pose2: {pose: [-0.542, -0.269, 0.0], orientation: [0.0, 0.0, 1.0, 0.018]} 

arm_pose:
  standby: [4.724388118992151, 3.836729629741731, 0.6832759890260246, 3.1519603397134714, 4.561281752765812, 0.5834928024609901]
  out: [3.5267376788565077, 3.9885852056269213, 0.8766939221474275, 1.4251207493772897, 4.336897647753403, 0.6887802769460275]
  drink_1: [4.318998009434763, 2.154619161493975, 4.905431940424609, 1.246678170539566, 4.641360876215622, 6.001312076347856]
  drink_2: [3.4867077045115265, 2.1402833653460784, 4.723771330883689, 0.4660301544204375, 4.250704705389098, 6.113903070385354]
  follow_1: [3.51177124616248, 2.8323336046365597, 4.792249724625114, 0.3388715360621516, 3.705581729228561, -0.26743204008369426]
  follow_2: [3.5338280789427348, 2.679259366236549, 5.1520039057429035, 0.3434728459250496, 3.9626558703055825, -0.2668717442798552]
  head_ready: [3.555024444374784, 2.925448102673981, 5.497535208743017, 0.4005828041841721, 4.015191782813466, -0.27277958432878935]

item_information:
  box: {arm_pose: [4.380617964420908, 3.8113491719210577, 0.6297706858813515, 1.9376650096067787, 4.466047112184578, 0.5653106693297194], height: 0.0, size: 0.6, id: 108}
